src/main/java/lotto/model/WinningLotto.java

```diff
@@ -1,24 +1,52 @@
 package lotto.model;
 
+import lotto.prize.LottoPrize;
+import lotto.prize.MatchInformation;
+
+import java.util.List;
 import java.util.Objects;
 
+import static java.util.stream.Collectors.toList;
+
 public class WinningLotto {
-    private final Lotto winningLotto;
+    private final Lotto winningNumber;
```

Comment:
```suggestion
    private final Lotto winningLotto;
```
이렇게 표현해주면 더 좋을 것 같아요!